[BUG]: Concurrent error occurred with select menu

### What happened?
Concurrent error occurred with select menu

### How can we reproduce issue?


### Relevant log output
net.dv8tion.jda.api.exceptions.ErrorResponseException: 10062: Unknown interaction
        at net.dv8tion.jda.api.exceptions.ErrorResponseException.create(ErrorResponseException.java:201)
        at net.dv8tion.jda.api.requests.Request.createErrorResponseException(Request.java:157)
        at net.dv8tion.jda.internal.requests.RestActionImpl.handleResponse(RestActionImpl.java:290)
        at net.dv8tion.jda.internal.requests.restaction.interactions.InteractionCallbackImpl.handleResponse(InteractionCallbackImpl.java:139)
        at net.dv8tion.jda.api.requests.Request.handleResponse(Request.java:304)
        at net.dv8tion.jda.internal.requests.Requester$WorkTask.handleResponse(Requester.java:442)
        at net.dv8tion.jda.internal.requests.Requester$WorkTask.access$300(Requester.java:378)
        at net.dv8tion.jda.internal.requests.Requester.execute(Requester.java:245)
        at net.dv8tion.jda.internal.requests.Requester.execute(Requester.java:157)
        at net.dv8tion.jda.internal.requests.Requester.execute(Requester.java:140)
        at net.dv8tion.jda.internal.requests.Requester$WorkTask.execute(Requester.java:406)
        at net.dv8tion.jda.api.requests.SequentialRestRateLimiter$Bucket.execute(SequentialRestRateLimiter.java:477)
        at net.dv8tion.jda.api.requests.SequentialRestRateLimiter$Bucket.run(SequentialRestRateLimiter.java:517)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: net.dv8tion.jda.api.exceptions.ContextException: null
        at net.dv8tion.jda.api.exceptions.ContextException.here(ContextException.java:54)
        at net.dv8tion.jda.api.requests.Request.<init>(Request.java:77)
        at net.dv8tion.jda.internal.requests.RestActionImpl.queue(RestActionImpl.java:203)
        at net.dv8tion.jda.internal.requests.restaction.interactions.InteractionCallbackImpl.queue(InteractionCallbackImpl.java:104)
        at net.dv8tion.jda.api.requests.RestAction.queue(RestAction.java:573)
        at net.dv8tion.jda.api.requests.RestAction.queue(RestAction.java:539)
        at com.movinfo.messenger.handler.ButtonClickHandler.sendMovieSelectMenu(ButtonClickHandler.java:58)
        at com.movinfo.messenger.handler.ButtonClickHandler.onButtonInteraction(ButtonClickHandler.java:23)
        at net.dv8tion.jda.api.hooks.ListenerAdapter.onEvent(ListenerAdapter.java:455)
        at net.dv8tion.jda.api.hooks.InterfacedEventManager.handle(InterfacedEventManager.java:98)
        at net.dv8tion.jda.internal.hooks.EventManagerProxy.handleInternally(EventManagerProxy.java:88)
        at net.dv8tion.jda.internal.hooks.EventManagerProxy.handle(EventManagerProxy.java:70)
        at net.dv8tion.jda.internal.JDAImpl.handleEvent(JDAImpl.java:169)
        at net.dv8tion.jda.internal.handle.InteractionCreateHandler.handleAction(InteractionCreateHandler.java:134)
        at net.dv8tion.jda.internal.handle.InteractionCreateHandler.handleInternally(InteractionCreateHandler.java:86)
        at net.dv8tion.jda.internal.handle.SocketHandler.handle(SocketHandler.java:39)
        at net.dv8tion.jda.internal.requests.WebSocketClient.onDispatch(WebSocketClient.java:1009)
        at net.dv8tion.jda.internal.requests.WebSocketClient.onEvent(WebSocketClient.java:892)
        at net.dv8tion.jda.internal.requests.WebSocketClient.handleEvent(WebSocketClient.java:870)
        at net.dv8tion.jda.internal.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:1048)
        at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:385)
        at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:276)
        at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:996)
        at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:755)
        at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108)
        at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64)
        at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45)
15:37:21.522 [ForkJoinPool.commonPool-worker-1] ERROR net.dv8tion.jda.api.requests.RestAction -- Encountered error while processing success consumer
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
        at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
        at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
        at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
        at java.base/java.util.Objects.checkIndex(Objects.java:361)
        at java.base/java.util.ArrayList.get(ArrayList.java:427)
        at com.movinfo.messenger.command.RoleManager.lambda$hasRole$2(RoleManager.java:96)
        at net.dv8tion.jda.api.requests.Request.lambda$onSuccess$0(Request.java:123)
        at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
15:37:23.155 [ForkJoinPool.commonPool-worker-1] ERROR net.dv8tion.jda.api.requests.RestAction -- Encountered error while processing success consumer
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
        at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
        at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
        at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
        at java.base/java.util.Objects.checkIndex(Objects.java:361)
        at java.base/java.util.ArrayList.get(ArrayList.java:427)
        at com.movinfo.messenger.command.RoleManager.lambda$hasRole$2(RoleManager.java:96)
        at net.dv8tion.jda.api.requests.Request.lambda$onSuccess$0(Request.java:123)
        at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)

### Environment

--- a/src/main/java/com/movinfo/messenger/command/RoleManager.java
+++ b/src/main/java/com/movinfo/messenger/command/RoleManager.java
@@ -93,8 +93,13 @@ public class RoleManager {
         guild.retrieveMember(user).queue(
             member -> {
                 if (member != null){
-                    Role role = guild.getRolesByName(roleName, true).get(0);
-                    future.complete(member.getRoles().contains(role));
+                    if (!guild.getRolesByName(roleName, true).isEmpty()){
+                        Role role = guild.getRolesByName(roleName, true).get(0);
+                        future.complete(member.getRoles().contains(role));
+                    } else {
+                        future.complete(false);
+                    }
+                    
                 } else {
                     future.complete(false);
                 }

